central/sensor/service/pipeline/nodeinventory/pipeline.go

```diff
@@ -63,12 +62,10 @@ func (p *pipelineImpl) Run(ctx context.Context, _ string, msg *central.MsgFromSe
 	if ninv == nil {
 		return errors.Errorf("unexpected resource type %T for node inventory", event.GetResource())
 	}
-	invStr := fmt.Sprintf("for node %s (id: %s)", ninv.GetNodeName(), ninv.GetNodeId())
-	log.Infof("received node inventory %s", invStr)
```

Comment:
Yes, there is a replacement for it:
- if all goes well, we will see `scanned inventory from node %q with %d components`
- in case of errors, we will see one of the errors

I moved this to debug, as otherwise the info would be printed twice.